consensus/mint-client/src/bin/main.rs

```diff
@@ -363,5 +363,25 @@ fn main() {
                 .write_json(&tx_file_path)
                 .expect("failed writing tx file");
         }
+
+        Commands::CheckSig {
+            signature,
+            hash,
+            pubkey,
+        } => {
+            // terminate with an exit code of 0 for success and 1 for failure
+            // allowing whoever started this binary to easily determine if submitting the
+            // transaction succeeded.
+            match pubkey.verify(&hash, &signature) {
+                Ok(()) => {
+                    println!("signature Ok");
```

Comment:
nit: it is slightly better to use log messages, so like 
```suggestion
                    log::info!(logger, "signature Ok");
```

Reply:
even for the output of a cli? This is supposed to output to stdout no matter the logging setting.

Reply:
most of the commands in mc-consensus-mint-client use println! to output their result.